Integratord unit tests failing

|Branch|Component|Source|
|--|--|--|
|4.9.0|Integratord|#23833|

We're getting the following report in the cmocka unit tests for _wazuh-integratord_:

```
[==========] Running 1 test(s).
[ RUN      ] test_OS_IntegratorD
[  ERROR   ] --- "Command ran successfully." != "Running script with args: integrations /tmp/virustotal-1111-2222.alert 123456    0 0 > /dev/null 2>&1"
unit_tests/wrappers/wazuh/shared/debug_op_wrappers.c:33: error: Check of parameter formatted_msg, function __wrap__mdebug1 failed
/root/wazuh-4.9.0/src/unit_tests/os_integrator/test_integrator.c:135: note: Expected parameter declared here
[   LINE   ] --- unit_tests/wrappers/wazuh/shared/debug_op_wrappers.c:33: error: Failure!
[  FAILED  ] test_OS_IntegratorD
[==========] 1 test(s) run.
[  PASSED  ] 0 test(s).
[  FAILED  ] 1 test(s), listed below:
[  FAILED  ] test_OS_IntegratorD
```

--- a/src/unit_tests/os_integrator/test_integrator.c
+++ b/src/unit_tests/os_integrator/test_integrator.c
@@ -132,8 +132,6 @@ void test_OS_IntegratorD(void **state) {
     will_return(__wrap_time, 1111);
     will_return(__wrap_os_random, 2222);
 
-    expect_string(__wrap__mdebug1, formatted_msg, "Running script with args: integrations /tmp/virustotal-1111-2222.alert 123456    0 0 > /dev/null 2>&1");
-
     will_return(__wrap_wpopenv, wfd);
 
     expect_value(__wrap_fgets, __stream, wfd->file_out);
@@ -173,8 +171,6 @@ void test_OS_IntegratorD(void **state) {
     expect_string(__wrap__mdebug2, formatted_msg, "File /tmp/pagerduty-1111-2222.options was written.");
 
     expect_fclose((FILE *)1, 0);
-
-    expect_string(__wrap__mdebug1, formatted_msg, "Running script with args: integrations /tmp/pagerduty-1111-2222.alert 123456   /tmp/pagerduty-1111-2222.options 0 0 > /dev/null 2>&1");
 
     will_return(__wrap_wpopenv, wfd);
 

